feature/funding/googleplay/src/main/kotlin/app/k9mail/feature/funding/googleplay/ui/contribution/ContributionList.kt

```diff
@@ -197,14 +209,33 @@ private fun ListEmptyView(
         verticalArrangement = Arrangement.spacedBy(MainTheme.spacings.double),
         modifier = modifier.padding(vertical = MainTheme.spacings.double),
     ) {
+        val context = LocalContext.current
+        val annotatedString = annotatedStringResource(
+            id = R.string.funding_googleplay_contribution_list_empty_message,
+            argument = buildAnnotatedString {
+                withStyle(
+                    style = SpanStyle(
+                        color = MainTheme.colors.primary,
+                        textDecoration = TextDecoration.Underline,
+                    ),
+                ) {
+                    append(stringResource(R.string.funding_googleplay_thunderbird_website_domain))
```

Comment:
The Compose APIs are changing constantly and I've seen there is a nicer way to achieve this now. Instead of adding a click listener on the whole text. You could use `withLink` to make just the link clickable.

```suggestion
                    withLink(
                        LinkAnnotation.Url(
                            url = stringResource(R.string.funding_googleplay_thunderbird_website_url),
                        ),
                    ) {
                        append(stringResource(R.string.funding_googleplay_thunderbird_website_domain))
                    }
```

Reply:
Great! I just was looking for something similar as we did with links and annotations before Compose but I implemented the other way to be aligned with other parts of the project.